tests/wiseagents/agents/test_phased_coordinator.py

```diff
@@ -9,6 +11,11 @@
 cond = threading.Condition()
 
 
+@pytest.fixture(scope="session", autouse=True)
+def run_after_all_tests():
+    WiseAgentRegistry.clear_agents()
+    WiseAgentRegistry.clear_contexts()
```

Comment:
If we want this to be run after all tests, we need to add `yield` as the first line. Anything before `yield` happens at the beginning and anything after `yield` will take place at the end of the test session.